test-app/types/test-app/index.d.ts

```diff
@@ -1 +1,7 @@
+import 'ember-source/types/stable';
 
+declare global {
```

Comment:
Do we need these global types anymore? Since we are not manually putting flatpickr on the window?